pkg/skaffold/build/docker/docker_test.go

```diff
@@ -135,7 +135,7 @@ func TestDockerCLIBuild(t *testing.T) {
 					test.err,
 				)
 				t.Override(&util.DefaultExecCommand, mockCmd)
-			} else if test.localBuild.UseBuildkit || test.localBuild.UseDockerCLI {
+			} else if (test.localBuild.UseBuildkit != nil && *test.localBuild.UseBuildkit) || test.localBuild.UseDockerCLI {
```

Comment:
This feels like it should be a `want` field in the test: `wantDockerCLI`.

Reply:
done.